three-cards-divination/src/main/resources/application.properties

```diff
@@ -12,3 +12,8 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.hibernate.ddl-auto=update
 ## Others
 spring.thymeleaf.check-template-location=false
+##Swagger
+springdoc.swagger-ui.path=/swagger-ui.html
+springdoc.api-docs.path=/api-docs
+# disable api-docs
+springdoc.api-docs.enabled=false
```

Comment:
**Suggestion:** It's a good practice to add a brief comment describing the purpose of each property. This will make it easier for other developers to understand the configuration. [best practice]
```suggestion
##Swagger
# Path for accessing the Swagger UI
springdoc.swagger-ui.path=/swagger-ui.html
# Path for accessing the Swagger Docs API
springdoc.api-docs.path=/api-docs
# Flag to enable/disable the Swagger Docs API
springdoc.api-docs.enabled=false
```